management-app/src/main/resources/static/css/main.css

```diff
@@ -0,0 +1,7 @@
+h1 {
```

Comment:
`main.css` and `style.css` files have the same contents. Consider using only one of them and delete the other.